controllers/component_build_controller_test.go

```diff
@@ -2347,4 +2261,99 @@ var _ = Describe("Component initial build controller", func() {
 		})
 	})
 
+	Context("Test pipeline selection", func() {
```

Comment:
This and previous test context look almost identical to me

Reply:
Updated.